src/runtime/virtcontainers/qemu.go

```diff
@@ -119,11 +119,11 @@ type qemu struct {
 }
 
 const (
-	consoleSocket = "console.sock"
-	qmpSocket     = "qmp.sock"
-	hmpSocket     = "hmp.sock"
-	vhostFSSocket = "vhost-fs.sock"
-	nydusdAPISock = "nydusd-api.sock"
+	consoleSocket      = "console.sock"
+	qmpSocket          = "qmp.sock"
+	extraMonitorSocket = "extra-monitor.sock"
```

Comment:
Just as a thought, do we need both `qmpSocket` and `extraMonitorSocket`? I understand that the latter is just a rename of an existing `hmpSocket` constant, however if I understand correctly, there will only ever be at most one monitor socket per a kata VM. If so, then we could either want to bake the name of the protocol used into the name of the socket file (which I could see value in, for debugging), in which case we'd need separate constants for `hmp.sock` and `qmp-pretty.sock`, or we don't want to do that, in which case it would make sense to use a single name (maybe `monitor.sock`?) for all protocol variants.

The currently proposed solution kind of doesn't want to make up its mind, using an explicit `qmp.sock` name for QMP but subsuming the pretty variant of QMP and HMP under `extra-monitor.sock`.

Reply:
This PR allows to add exactly one monitor socket. This socket must have a different _address_ than the primary socket and the primary socket is de facto a plain QMP session. We thus cannot `bake the name of the protocol used into the name of the socket file` since the extra socket could be plain QMP as well.
Not sure if the protocol variant really needs to be part of the extra socket name because the user already knows what he asked for in the configuration TOML... or do you have some other scenario in mind @pmores ?

Reply:
Alright, so it seems I misunderstood, I apologize. If it's the case that there will (almost) always be a `qmpSocket` and independent of that there _might_ be an `extraMonitorSocket` if requested by `configuration.toml` then please ignore my previous comment.